packages/fxa-settings/src/pages/Signin/SigninRecoveryMethod/Phone/ConfirmRecoveryCode/en.ftl

```diff
@@ -0,0 +1,12 @@
+## ConfirmRecoveryCode page
+
+confirm-recovery-code-flow-heading = Sign in
+confirm-recovery-code-with-code-heading = Enter recovery code
+
+# Text that explains the user should check their phone for a recovery code
+# $phoneNumber - The users phone number
+confirm-recovery-code-with-code-instruction = A six-digit code was sent to <span>{ $phoneNumber }</span> by text message. This code expires after 5 minutes.
```

Comment:
I think this needs to be `maskedPhoneNumber`, not `phoneNumber`:

![image](https://github.com/user-attachments/assets/9d581529-6848-4083-8f94-630acc6c74ab)